pymatgen_diffusion/neb/pathfinder.py

```diff
@@ -290,25 +312,68 @@ def __init__(self, isite, esite, symm_structure):
         """
         self.isite = isite
         self.esite = esite
+        self.iindex = None
+        self.eindex = None
         self.symm_structure = symm_structure
         self.msite = PeriodicSite(
-            esite.specie,
-            (isite.frac_coords + esite.frac_coords) / 2, esite.lattice)
+            esite.specie, (isite.frac_coords + esite.frac_coords) / 2, esite.lattice
+        )
         sg = self.symm_structure.spacegroup
         for i, sites in enumerate(self.symm_structure.equivalent_sites):
             if sg.are_symmetrically_equivalent([isite], [sites[0]]):
                 self.iindex = i
             if sg.are_symmetrically_equivalent([esite], [sites[0]]):
                 self.eindex = i
 
+        # if no index was identified then loop over each site until something is found
```

Comment:
All the new logic is in this block everything else is just reformatting by pre-commit.yaml + black